src/main/features/claimant-response/form/models/settlementAgreement.ts

```diff
@@ -0,0 +1,31 @@
+import { IsDefined } from 'class-validator'
+import { IsBooleanTrue } from '@hmcts/cmc-validators'
+
+export class ValidationErrors {
+  static readonly SETTLEMENT_AGREEMENT_REQUIRED_MESSAGE: string = 'Please select I confirm I’ve read and accept the terms of the agreement.'
```

Comment:
Could be extracted to either common or settlement agreement validation errors as its the same as the other one
REC